config/path

```diff
@@ -200,8 +200,12 @@ if [ -z "$PATH" -o "$PATH" = "${PATH#$ROOT/$TOOLCHAIN/bin:}" ]; then
   export PATH="$ROOT/$TOOLCHAIN/bin:$ROOT/$TOOLCHAIN/sbin:$PATH"
 fi
 
-if [ -z "$LD_LIBRARY_PATH" -o "$LD_LIBRARY_PATH" = "${LD_LIBRARY_PATH#$ROOT/$TOOLCHAIN/lib:}" ]; then
-  export LD_LIBRARY_PATH="$ROOT/$TOOLCHAIN/lib:$LD_LIBRARY_PATH"
+if [ -z "$LD_LIBRARY_PATH" ]; then
+  export LD_LIBRARY_PATH="$ROOT/$TOOLCHAIN/lib"
+else
+  if [ "$LD_LIBRARY_PATH" = "${LD_LIBRARY_PATH#$ROOT/$TOOLCHAIN/lib:}" ]; then
```

Comment:
This would be better as an `elif`

And now that the trailing : is no longer added in line 204, surely the comparison is incorrect? Shouldn't it now be:
```
[ "$LD_LIBRARY_PATH" = "${LD_LIBRARY_PATH#$ROOT/$TOOLCHAIN/lib}" ]; then
```
as there is no guarantee that a colon will be present.

An alternative would be:
```
elif ! [[ $LD_LIBRARY_PATH =~ ^$ROOT/$TOOLCHAIN/lib ]]; then
```
which might be easier to understand.